### DISCUSSION TITLE
Explanantion of GeochemData file

### DISCUSSION THREAD

**Discussion by Santhosh91000** [GeoChemFoam/GeoChemFoam#22] [Q&A] [answered]
Hello all,

Hope all is well! 
I just wanted to know if there is anywhere qa description of the chemcal data file for interReactiveTransferFoam ? In the tutorial case (2DCavity). The chemical database looks like this:

# PHREEQC.DAT for calculating pressure dependence of reactions, with
#   molal volumina of aqueous species and of minerals, and
#   critical temperatures and pressures of gases used in Peng-Robinson's EOS.
# Details are given at the end of this file.

SOLUTION_MASTER_SPECIES
#
#element	species	alk	gfw_formula	element_gfw
#
H		H+	-1.0	H		1.008
H(0)		H2	0	H
H(1)		H+	-1.0	0
E		e-	0	0.0		0
O		H2O	0	O		16.0
O(0)		O2	0	O
O(-2)		H2O	0	0
C		CO3-2	2.0	HCO3		12.0111
C(+4)		CO3-2	2.0	HCO3
Alkalinity	CO3-2	2.0	61.0173 	61.0173

SOLUTION_SPECIES
H+ = H+
	-gamma	9.0	0

	-dw	 9.31e-9
e- = e-
H2O = H2O
CO3-2 = CO3-2
	-gamma	5.4	0
	-dw	 0.955e-9
	-Vm  5.95  0  0  -5.67  6.85  0  1.37  106  -0.0343  1 # ref. 1
# aqueous species
H2O = OH- + H+
	-analytic  293.29227  0.1360833  -10576.913  -123.73158  0  -6.996455e-5
	-gamma	3.5	0
	-dw	 5.27e-9
	-Vm  -9.66  28.5  80.0 -22.9 1.89 0 1.09 0 0 1 # ref. 1
2 H2O = O2 + 4 H+ + 4 e-
	-log_k	-86.08
	-delta_h 134.79 kcal
	-dw	 2.35e-9
	-Vm  5.7889  6.3536  3.2528  -3.0417  -0.3943 # supcrt
2 H+ + 2 e- = H2
	-log_k	-3.15
	-delta_h -1.759 kcal
	-dw	 5.13e-9
	-Vm 6.52  0.78  0.12 # supcrt
CO3-2 + H+ = HCO3-
	-log_k	10.329
	-delta_h -3.561	kcal
	-analytic	107.8871	0.03252849	-5151.79	-38.92561	563713.9
	-gamma	5.4	0
	-dw	 1.18e-9
	-Vm  8.472  0  -11.5  0  1.56  0  0  146  3.16e-3  1 # ref. 1
CO3-2 + 2 H+ = CO2 + H2O
	-log_k	16.681
	-delta_h -5.738	kcal
	-analytic	464.1965	0.09344813	-26986.16	-165.75951	2248628.9
	-dw	 1.92e-9
	-Vm   7.29  0.92  2.07  -1.23  -1.60 # ref. 1 + McBride et al. 2015, JCED 60, 171
2CO2 = (CO2)2 # activity correction for CO2 solubility at high P, T
	-log_k -1.8
	-analytical_expression  8.68  -0.0103  -2190
	-Vm   14.58  1.84  4.14  -2.46  -3.20

	-delta_h -0.396	kcal
	-Vm  6.16  0  29.4  0  0.9 # ref. 2
END

But no information is given about the variables. I can guess that Vm is molar volum maybe and log_k related to equilibrium constant. Maybe analytics is for activities values ? But I am not too sure. Can anyone help me find or give good description about the varibales in this file.

Thanks a lot for the time!

Sincerely,
Santhosh

**Comment by YuxuanZZ** [comment, marked as answer]
I'm using examples in this book to understand phreeqc's code.(https://pubs.usgs.gov/publication/tm6A43) This book is from phreeqc's official website.

**Reply by Santhosh91000** [reply]
Hello YuxuanZZ,

Thanks a lot for the fast reply! I'll check it out.

Sincerely,
Santhosh
